### PR TITLE
Tests - Single additional test to cover properties on gridd2drepresentation objects

### PR DESCRIPTION
**Pull request by emmanesbit**
Single test to cover property collections with a grid2drepresentation as a support. Properties with indexable of cells and nodes added for testing.

### CODE REVIEW

**Review by cflynn3 — APPROVED** [review]

File: tests/test_property.py [hunk 1]
```diff
@@ -2036,3 +2036,63 @@ def test_remove_cached_from_imported_list(example_model_with_properties):
...
+    pc = rqp.PropertyCollection()
+    pc.set_support(support = mesh)
+
+    # Assert
+    assert pc.support is not None
+    assert isinstance(pc.support, rqs.Mesh)
+    assert pc.support == mesh
+    assert pc.support_uuid == mesh.uuid
+
+    # Act - add different indexable element properties to collection
```
Comment (cflynn3) [inline tests/test_property.py:2077]: Ideally these would be separate tests, as they represent separate test cases.

File: tests/test_property.py [hunk 2]
```diff
@@ -2036,3 +2036,63 @@ def test_remove_cached_from_imported_list(example_model_with_properties):
...
+                                         indexable_element = 'nodes')
+    pc.write_hdf5_for_imported_list()
+    pc.create_xml_for_imported_list_and_add_parts_to_model()
+
+    # Assert
+    for part in pc.parts():
+        if pc.citation_title_for_part(part) == 'TESTnode':
+            assert pc.indexable_for_part(part) == 'nodes'
+        else:
+            assert pc.indexable_for_part(part) == 'cells'
```
Comment (andy-beer) [inline tests/test_property.py:2098]: Could be good to put in a few more assertions. For example to call the pc.cached_part_array_ref() method and check the shape (and maybe the values) of the array.

**Review by andy-beer — COMMENTED** [review]
This is good Emma – could you add a few more assertions at the end there please.

**Review by andy-beer — APPROVED** [review]
Ready to merge

**Pull request merged**